content/applications/sales/sales/products_prices/loyalty_discount.rst

```diff
@@ -32,43 +32,51 @@ Or, if there are already existing programs, select an existing program to edit i
    :alt: Discount and loyalty program template cards.
 
 .. note::
-   Templates only appear when no programs have been created, and they disappear once the first
+   Templates **only** appear when no programs have been created, and they disappear once the first
    program is created.
 
 Creating or editing a program opens the program form.
 
-The program form contains the following options:
+.. image:: loyalty_discount/price-programs.png
+   :align: center
+   :alt: Program options on the loyalty program form.
 
-- :guilabel:`Program Name`: Enter the name of the program. The program name is *not* visible to the
+The program form contains the following fields:
+
+- :guilabel:`Program Name`: enter the name of the program. The program name is *not* visible to the
   customer.
-- :guilabel:`Program Type`: Select the desired :ref:`program type
+- :guilabel:`Program Type`: select the desired :ref:`program type
   <sales/pricing_management/program-types>`.
-- :guilabel:`Currency`: Select the currency used for the program.
-- :guilabel:`Points Unit`: Enter the name of the points used for the :guilabel:`Loyalty Cards`
+- :guilabel:`Currency`: select the currency used for the program.
+- :guilabel:`Pricelist`: if desired, select a pricelist from the drop-down menu to have this loyalty
+  program applied to a specific pricelist (and customers attached to the pricelist). More than one
+  pricelist can be selected in this field. When a single loyalty program is linked to several
+  pricelists, it makes it viable for different customer segments to have different pricelists, but
+  the *same* loyalty programs. If this field is left blank, the program applies to everyone,
+  regardless of pricelist.
+- :guilabel:`Points Unit`: enter the name of the points used for the :guilabel:`Loyalty Cards`
   program (e.g. `Loyalty Points`). The points unit name *is* visible to the customer. This field is
   only available when the :guilabel:`Program Type` is set to :guilabel:`Loyalty Cards`.
-- :guilabel:`Validity`: Select the date until which the program is valid. Leave this field blank for
-  no end date, meaning the program is always valid and does not expire.
-- :guilabel:`Limit Usage`: Check this box and enter a number to limit the number of times the
-  program can be used during the :guilabel:`Validity` period.
-- :guilabel:`Company`: In the case of multiple companies, choose the company for which the program
+- :guilabel:`Start Date`: select the date on which the program becomes valid. Leave this field blank
+  if the program should always be valid and not expire.
```

Comment:
```suggestion
  if the program never expires.
```
Just for brevity.

Reply:
I fully appreciate this suggestion, and understand the desire for brevity. However, with this suggestion, I fear that a reader would misunderstand, and possibly think (incorrectly) that this program would *never be ABLE* to expire with this configuration in place...hence, my slightly elongated version...

just trying to make sure the reader fully understands the concept I am trying to convey. And, since we're talking about a date range (or lack thereof, in this case), I think going the "extra mile" with some additional words to explain the concept is more beneficial (though, not as brief).